pom.xml

```diff
@@ -33,6 +33,26 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.2.19</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>2.4.3</version>
```

Comment:
Większośc tego co zaczyna się od spring-boot-starter to pakiet bibliotek bazujący na spring-boot-starter-parent i z niego bierze wersję. Dlatego w takich przypadakch nei dodajem version w dependency. Gdy będziemy podnosić wersję spring boota to podniesiemy tylko wersję parenta a nie każdego dependecy